src/render/attrs/index.js

```diff
@@ -10,6 +10,42 @@ const attrsNotToRender = {
   dangerouslySetInnerHTML: true
 };
 
+const attrsWithExplicitBoolValue = {
+  "aria-expanded": true,
+  "aria-haspopup": true
+};
```

Comment:
Strictly speaking, the only public APIs of reactDomServer are `renderToString`  and `renderToStaticMarkup`. 

Relying on `DOMProperty` indeed has the downside of relying on some react internal that is subject to change. However, any change in the internal of React would most likely also affect the result of `renderToString` so Rapscallion needs to be tied with specific versions of React anyways to have a consistent behavior.

The good side is that it is in the `shared` folder and used by several different renderers, so it should not be ditched soon I believe.

Some other React libraries rely on it like https://github.com/chromakode/react-html-email for similar use case.